llzk/src/dialect/bool/attrs.rs

```diff
@@ -0,0 +1,76 @@
+use llzk_sys::{llzkAttributeIsAFeltCmpPredicateAttr, llzkFeltCmpPredicateAttrGet, LlzkCmp};
+use melior::{
+    ir::{Attribute, AttributeLike},
+    Context,
+};
+use mlir_sys::MlirAttribute;
+
+/// Possible options for creating [`CmpPredicateAttribute`].
+#[repr(u32)]
+pub enum CmpPredicate {
+    Eq = llzk_sys::LlzkCmp_LlzkCmp_EQ,
+    Ne = llzk_sys::LlzkCmp_LlzkCmp_NE,
+    Lt = llzk_sys::LlzkCmp_LlzkCmp_LT,
+    Le = llzk_sys::LlzkCmp_LlzkCmp_LE,
+    Gt = llzk_sys::LlzkCmp_LlzkCmp_GT,
+    Ge = llzk_sys::LlzkCmp_LlzkCmp_GE,
+}
+
+/// Attribute representing a comparison predicate.
+pub struct CmpPredicateAttribute<'c> {
```

Comment:
Q: Any reason we need `CmpPredicateAttribute`? It seems like a think wrapper around `Attribute`?